lib/ruby/truffle/bin/truffleruby

```diff
@@ -0,0 +1 @@
+../../../../bin/truffleruby
```

Comment:
But I don't want to ship something that refers to `mx` - that's just dead code in a distribution.